<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gpkg2FusionDialog</class>
 <widget class="QDialog" name="Gpkg2FusionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPKG ‚Üí Fusion</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">

   <!-- Columna izquierda: opciones -->
   <item>
    <layout class="QVBoxLayout" name="leftLayout">

     <!-- Carpeta de entrada -->
     <item>
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>üìÇ Carpeta de entrada (GeoPackages a fusionar):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="inputLayout">
       <item>
        <widget class="QLineEdit" name="inputFolderLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="inputBrowseButton">
         <property name="text">
          <string>Explorar...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- Archivo de salida -->
     <item>
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>üíæ Archivo de salida (GeoPackage fusionado):</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="outputLayout">
       <item>
        <widget class="QLineEdit" name="outputFileLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="outputBrowseButton">
         <property name="text">
          <string>Guardar como...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <!-- Log de ejecuci√≥n -->
     <item>
      <widget class="QLabel" name="logLabel">
       <property name="text">
        <string>üìú Registro de ejecuci√≥n:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="logTextEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Mensajes de ejecuci√≥n...</string>
       </property>
      </widget>
     </item>

     <!-- Botones -->
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <item>
        <widget class="QPushButton" name="runButton">
         <property name="text">
          <string>‚ñ∂ Ejecutar fusi√≥n</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>‚ùå Cancelar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

    </layout>
   </item>

   <!-- Columna derecha: descripci√≥n + advertencias -->
   <item>
    <layout class="QVBoxLayout" name="rightLayout">
     <item>
      <widget class="QTextEdit" name="descriptionText">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>
         <![CDATA[
         <h1><b>gpkg2fusion</b></h1>
         <p><b>Descripci√≥n de la herramienta:</b></p>
         <p>Fusiona todos los archivos <b>GeoPackage (.gpkg)</b> de la carpeta seleccionada (y de sus subcarpetas) 
            en un √∫nico archivo GeoPackage de salida.</p>
         <ul>
           <li><b>Carpeta de entrada:</b> donde est√°n los archivos GPKG a fusionar.</li>
           <li><b>Archivo de salida:</b> ruta del GeoPackage resultante.</li>
         </ul>
         <p><b>‚ö† Advertencias:</b></p>
         <ul>
           <li>Si el archivo de salida ya existe, ser√° sobrescrito.</li>
         </ul>
         ]]>
        </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
